benchmarks/data-caching/client/Dockerfile

```diff
@@ -6,18 +6,16 @@ RUN groupadd -r memcache && useradd -r -g memcache memcache
 
 ENV DEBIAN_FRONTEND noninteractive
 
-COPY memcached.tar.gz /memcached.tar.gz
+# libevent-dev will include libevent, so it's not necessary to install it standalone
+# RUN apt-get update && apt-get install -y libevent-2.1-6 vim --no-install-recommends \
+	# && rm -rf /var/lib/apt/lists/*
 
-RUN apt-get update && apt-get install -y libevent-2.1-6 vim --no-install-recommends \
-	&& rm -rf /var/lib/apt/lists/*
-
-RUN buildDeps='curl gcc libc6-dev libevent-dev make' \	
+RUN buildDeps='curl gcc libc6-dev libevent-dev make git vim' \	
 	&& set -x \
 	&& apt-get update && apt-get install -y $buildDeps --no-install-recommends \
 	&& rm -rf /var/lib/apt/lists/* \
-	&& mkdir -p /usr/src/memcached \
-	&& tar -xzf memcached.tar.gz -C /usr/src/memcached --strip-components=1 \
-	&& rm memcached.tar.gz \
+	&& git config --global http.sslverify false \
```

Comment:
You don't need to use git to fetch the files. You could just download the zip version of the repo.
I also just created a release so you can just simply download that release tarball.